project/AppDependencies.scala

```diff
@@ -3,26 +3,25 @@ import sbt.*
 object AppDependencies {
   import play.core.PlayVersion
 
-  val HmrcMongoPlayVersion              = "1.3.0"
+  val HmrcMongoPlayVersion              = "1.4.0"
   val PlayFrontendHmrcVersion           = "7.27.0-play-28"
   val PlayConditionalFormMappingVersion = "1.13.0-play-28"
   val BootstrapFrontendPlayVersion      = "7.23.0"
   val CatsVersion                       = "2.10.0"
 
-  val ScalaTestVersion          = "3.2.10"
```

Comment:
dont see any benefit in having variables for version unless you are using it at least twice